lib/ddtrace.rb

```diff
@@ -1,31 +1,11 @@
 # Load tracing
 require 'datadog/tracing'
```

Comment:
Definitely not for this PR, but would it make sense to rename this file to `lib/datadog.rb` before 1.0.0?

Or is there a deeper reason for still having a few things under the `ddtrace` name?

Reply:
There's a deeper reason: main motivation is future plans for an upgrade path. By keeping `require 'ddtrace'` as its own thing, it means we can later introduce a `gem 'datadog'` with its own `require 'datadog'` without creating conflict with this gem.

Besides, there's something conventional and nice about having a file named after the package that contains it as a main entry point. Since our package is named `ddtrace`, it makes sense for this to be called `require 'ddtrace'`.